integrated_channels/integrated_channel/transmitters/content_metadata.py

```diff
@@ -228,6 +228,13 @@ def _transmit_action(self, content_metadata_item_map, client_method, action_name
                     transmission.api_response_status_code = response_status_code
                     was_successful = response_status_code < 300
                     api_content_response = self._filter_api_response(response_body, content_id)
+                    (api_content_response, was_truncated) = truncate_string(api_content_response)
+                    if was_truncated:
+                        self._log_info(
+                            f'integrated_channel_content_transmission_id={transmission.id}, '
+                            f'api response truncated',
+                            course_or_course_run_key=content_id
+                        )
```

Comment:
I think we can remove this log, won't help us capture anything that we aren't already capturing through other logs.

Reply:
it was suggested by @zamanafzal to observe the frequency of response exceeding the defined max limit so that we can update the max limit in the future accordingly.

Reply:
Makes sense.